clearpath_config/platform/attachments/generic.py

```diff
@@ -25,21 +25,10 @@
 # CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
-# Generic Robot Platform Configuration
-from clearpath_config.common.types.config import BaseConfig
 from clearpath_config.common.types.platform import Platform
-from clearpath_config.platform.attachments.config import BaseAttachmentsConfig
+from clearpath_config.platform.types.attachment import PlatformAttachment
 
 
-# Generic Attachments Configuration
-class GENERICAttachmentsConfig(BaseAttachmentsConfig, BaseConfig):
+# Generic Attachments
+class GENERICAttachment(PlatformAttachment):
```

Comment:
Lets call this `GenericAttachment` to match other naming.

Reply:
The thing is GENERIC is the "code" like A200.